metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -2103,6 +2125,22 @@ class MetalsLanguageServer(
     }
   }
 
+  @JsonRequest("workspace/willRenameFiles")
+  def willRenameFiles(
+      params: RenameFilesParams
+  ): CompletableFuture[WorkspaceEdit] =
+    CancelTokens.future { _ =>
```

Comment:
As Scala Futures are uncancellable, there is no cancellation logic. At least we might want to somehow signal about long-running computations here /shrug/